test/functional/sessions.test.js

```diff
@@ -194,4 +196,44 @@ describe('Sessions', function () {
 
     generateTopologyTests(testSuites, testContext, testFilter);
   });
+
+  context('unacknowledged writes', () => {
+    it('should not include session for unacknowledged writes', {
```

Comment:
this test needs the same `mongodb: >= 3.6` otherwise we're just ensuring sessions aren't added in these unsupported versions

Reply:
Add the version to this one test, and I updated all tests to use `>=3.6.0` and NOT `>3.6.0`.